### DISCUSSION TITLE
globs for --workspaces option

### DISCUSSION THREAD

**Discussion by trusktr** [npm/feedback#577] [CLI] [answered]
It would be nice if we had some sort of pattern matching, being able to pass a pattern to the `--workspaces` option thus being able to more easily run commands in subsets of packages. f.e. `npm run build --workspaces "./pkgs/test-*"`

**Comment by MylesBorins** [comment, marked as answer]
@trusktr if you would like to suggest new features to the CLI the best course of action is to open up an issue over at https://github.com/npm/rfcs
